core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -364,6 +364,25 @@ def show_warning(
     )
 
 
+def show_danger(
+    br_name: str,
+    content: str,
+    content_short: str | None = None,
+    value: str | None = None,
+    title: str | None = None,
+    verb_cancel: str | None = None,
+    br_code: ButtonRequestType = ButtonRequestType.Warning,
+) -> Awaitable[None]:
+    if content_short is None:
+        content_short = content
```

Comment:
apart from the previous comment, why showing `content_short` on `TT`?

Reply:
Because that's what Figma says ;)